content/applications/finance/fiscal_localizations/uruguay.rst

```diff
@@ -0,0 +1,321 @@
+- :guilabel:`DGI Main Branch Code`:
+
+After configuring the company in the database settings, navigating to :menuselection:`Contacts` and
+search for your company to verify the following:
+
+- Company type is set to :guilabel:`Company`.
+- Verify the :guilabel:`Identification Number` :guilabel:`Type` is :guilabel:`RUT / RUC`.
+
+.. _l10n_uy/uruware-account:
+
+Set up Uruware account
+----------------------
+
+To generate electronic documents in Odoo, a Uruware account is necessary.
+
+To set up a Uruware account, first be sure to have a valid Odoo subscription. Then follow the steps
+proceeding steps:
+
+Locate the Uruware credentials settings by navigating to the :menuselection:`Accounting -->
+Configuration --> Settings: Uruguay Localization section`. Take a note of the admin email address
+that is set, this will be the same email used in Uruware. Click on :guilabel:`Create Uruware
+Account`.
+
+.. note::
+   This action will create an account with Uruware with the following information:
+
+   - Legal name (razón social)
+   - RUT from the company
+   - Administrator email
+   - User name and last name
+   - Odoo database link
+
+.. important::
+   Add in any of the information that is missing to create your account.
+
+Once the account is created, a confirmation email is sent to the email used to create the account,
+containing the account credentials to configure the account directly in the Uruware portal:
+
+- Testing portal: https://odootest.ucfe.com.uy/Gestion/
+- Production portal: https://prod6109.ucfe.com.uy/Gestion/
+
+Use the account credentials in the email to log in to the portal.
+
+Electronic invoice data
+-----------------------
+
+To configure the electronic invoice data, an environment and credentials need to be configured. To
+do so, navigate to :menuselection:`... --> ...`.
+
+First, select the :guilabel:`UCFE Web Services` environment:
+
+- :guilabel:`Production`: for production databases. In this mode, electronic documents are sent to
+  |DGI| through Uruware for their validation.
+- :guilabel:`Testing`: for test databases. In this mode, the direct connection flows can be tested,
+  with the files sent to the |DGI| testing environment through Uruware.
+- :guilabel:`Demo`: files are created and accepted automatically in demo mode but are **not** sent
+  to the |DGI|. For this reason, rejection errors will not appear in this mode. Every internal
+  validation can be tested in demo mode. Avoid selecting this option in a production database.
+
+Then, enter the :guilabel:`Uruware Credentials`:
+
+- :guilabel:`Uruware WS Password`
+- :guilabel:`Commerce Code`
+- :guilabel:`Terminal Code`
+
+.. image:: uruguay/electronic-invoice-data.png
+   :alt: Required information for electronic invoice.
+   :align: center
+
+.. note::
+   These credentials can be obtained from the email received when the :ref:`Uruguay account was
+   created <l10n_uy/uruware-account>`.
```

Comment:
Not really, and actually we should be careful, the credentials they receive in the email will be to configure their account in Uruware, and in Uruware they will get these credentials (Uruware WS password, commerce code and terminal code).